### ISSUE TITLE
Save MTD bias potentials for later runs

### ISSUE THREAD

**Issue by matheger** [grimme-lab/crest#43]
When running multiple crest ensemble calculations on the same molecule, each run essentially starts from zero, with no biasing in the metadynamics. Would it make sense to have the option to save the bias potentials added in a crest run to be reused in future runs?

**Comment by pprcht** [comment]
Yes, that idea (in principle) works. In fact, we used this in http://dx.doi.org/10.1039/d1sc00621e where this approach is called static metadynamics (sMTD) and is conceptionally similar to umbrella sampling.
The main difficulty is that many MDs will blow up right at the beginning if too many bias structures are used from read-in input. It's a lot of energy that enters the simulation and compared to the regular MTD there is not much time for relaxation of the system.

Currently we don't have external routines for reading in a saved bias, but that is something interesting to work on. The bias would be saved simply as an ensemble of structures.
